enumerable.rb

```diff
@@ -0,0 +1,158 @@
+# rubocop:disable Layout/LineLength
+# rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable  Metrics/MethodLength
+# rubocop:disable  Metrics/ModuleLength
+module Enumerable
+  def my_each
+    return to_enum(:my_each) unless block_given?
+
+    n = length
+    arr = *self
+    n.times do |i|
+      arr[i] = yield(arr[i])
+    end
+    self
+  end
```

Comment:
Good work implementing some of the my_each functionalities. However, there are some changes to be made :arrow_down: 

- [ ]  when a block is given when self is a range it should return the range itself after calling the given block once for each element in self

for example: if you have a range.my_each(&block) it should equal range.my_each(&block)

kindly reference here for the behaviour of the each method [link](https://apidock.com/ruby/Range/each)  `(10..15).each {|n| print n, ' ' }
# prints: 10 11 12 13 14 15`